docs/UserGuide.md

```diff
@@ -341,15 +341,15 @@ ____________________________________________________________
 
 #### 3.1.10 Saving and exiting: `bye` <a name="bye"></a>
 
-This command will save all the appplications
+This command will save all the applications
```

Comment:
Should it be "save all the application data" instead?

Reply:
yea sry i changed it in my current version alr